---------------------
PatchSet 328 
Date: 2010/10/02 12:48:12
Author: millert
Branch: HEAD
Tag: (none) 
Log:
Suggest modern TIOCGSIZE, not the old sgtty TIOCGWINSZ.
On OpenBSD they are identical but new code should use the new name.

Members: 
	pod/perlfaq8.pod:1.12->1.13 

Index: src/gnu/usr.bin/perl/pod/perlfaq8.pod
diff -u src/gnu/usr.bin/perl/pod/perlfaq8.pod:1.12 src/gnu/usr.bin/perl/pod/perlfaq8.pod:1.13
--- pod/perlfaq8.pod:1.12	Fri Sep 24 15:07:04 2010
+++ pod/perlfaq8.pod	Sat Oct  2 19:48:12 2010
@@ -231,10 +231,10 @@
 illustrative:
 
 	require 'sys/ioctl.ph';
-	die "no TIOCGWINSZ " unless defined &TIOCGWINSZ;
+	die "no TIOCGSIZE " unless defined &TIOCGSIZE;
 	open(TTY, "+</dev/tty")                     or die "No tty: $!";
-	unless (ioctl(TTY, &TIOCGWINSZ, $winsize='')) {
-		die sprintf "$0: ioctl TIOCGWINSZ (%08x: $!)\n", &TIOCGWINSZ;
+	unless (ioctl(TTY, &TIOCGSIZE, $winsize='')) {
+		die sprintf "$0: ioctl TIOCGSIZE (%08x: $!)\n", &TIOCGSIZE;
 	}
 	($row, $col, $xpixel, $ypixel) = unpack('S4', $winsize);
 	print "(row,col) = ($row,$col)";
